src/dawn/engine_dawn.cc

```diff
@@ -932,8 +933,29 @@ Result EngineDawn::DoPatchParameterVertices(
   return Result("Dawn:DoPatch not implemented");
 }
 
-Result EngineDawn::DoBuffer(const BufferCommand*) {
-  return Result("Dawn:DoBuffer not implemented");
+Result EngineDawn::DoBuffer(const BufferCommand* command) {
+  Result result;
+
+  // TODO(SarahM0): Make this work for compute pipeline
```

Comment:
If this does not support compute pipeline yet, just return error message for compute pipeline here?

```
if (command->GetPipelineType == Compute) // fix this correctly
return Result("Compute pipeline is not supported by Dawn backend");
```